guides/common/modules/proc_deploying-the-ca-certificate-on-a-host-manually.adoc

```diff
@@ -0,0 +1,63 @@
+[id="deploying-the-ca-certificate-on-a-host-manually"]
+= Deploying the CA certificate on a host manually
```

Comment:
I do not see us updating the CA certificate stored under `/etc/rhsm/` for use by subscription-manager.